jaxrs-api/src/main/java/jakarta/ws/rs/core/Response.java

```diff
@@ -67,6 +67,11 @@ protected Response() {
 
     /**
      * Get the complete status information associated with the response.
+     * <p>
+     * Comparing {@link StatusType} directly might unexpectedly return {@code false}.
+     *
+     * <b>Instead compare {@code statusCode} directly</b>.
```

Comment:
```suggestion
     * Instead compare {@code statusCode} directly.
```

IMO this shouldn't be bold.